.github/workflows/gh-ci.yaml

```diff
@@ -212,8 +212,16 @@ jobs:
     - name: deploy_docs
       if: github.event_name != 'pull_request'
       run: |
-        # place the deploy call here
-        echo "Oh, maple syrup roast parsnips [Richard Gowers]"
+        python -c 'import os,sys,fcntl; flags = fcntl.fcntl(sys.stdout, fcntl.F_GETFL); fcntl.fcntl(sys.stdout, fcntl.F_SETFL, flags&~os.O_NONBLOCK);'
+        cd package/MDAnalysis
+        export VERSION=$(python -c "import version; print(version.__version__)")
+        cd -
+        export GH_DOC_BRANCH=$GITHUB_REF
+        export GH_TOKEN=${{ secrets.GITHUB_TOKEN }}
```

Comment:
Does GH-actions obscure encrypted tokens in the logs?

Reply:
Is this the old oauth2 token or a new one?

Reply:
The [``secrets.GITHUB_TOKEN`` is automatically created by GH](https://docs.github.com/en/free-pro-team@latest/actions/reference/authentication-in-a-workflow#about-the-github_token-secret). But, [no](https://docs.github.com/en/free-pro-team@latest/actions/reference/encrypted-secrets#limits-for-secrets), Github does not redact secrets if they are printed.

Reply:
So yeah this could be [an issue](https://github.community/t/how-to-see-my-git-secrets/123668/5) if someone is really determined to get it and knows how to google "how to see git secrets"

Reply:
Update: Yes, you can echo `{{ secrets.GITHUB_TOKEN }}`, but it's a) [single-use](https://docs.github.com/en/free-pro-team@latest/actions/reference/authentication-in-a-workflow#about-the-github_token-secret) and b) read-only on forked repositories. This PR from my [alt account](https://github.com/lilyminium/mdanalysis/runs/1491001169#step:7:2796) fails to write to lilyminium/mdanalysis:gh-pages with a permission error.